flink-table/flink-table-common/src/main/java/org/apache/flink/table/types/logical/LocalZonedTimestampType.java

```diff
@@ -72,7 +72,8 @@
                     java.time.Instant.class.getName(),
                     Integer.class.getName(),
                     Long.class.getName(),
-                    TimestampData.class.getName());
+                    TimestampData.class.getName(),
+                    java.sql.Timestamp.class.getName());
```

Comment:
Are we still planning to support `LocalDateTime` as well? Or is `java.sql.Timestamp` enough for us?  Supporting `LocalDateTime` would mean that we need access to configuration in converters, correct?

Reply:
yes, we think `java.sql.Timestamp` is enough, Supporting `LocalDateTime` is an impossible mission which means we need pass configuration to converters, but some place we're hard to get the configuration eg. format , serializers